tests/meta/test_dynamic_weighted_majority.py

```diff
@@ -0,0 +1,45 @@
+from skmultiflow.meta import DynamicWeightedMajority
+from skmultiflow.data import SEAGenerator
+from skmultiflow.bayes import NaiveBayes
+import numpy as np
+
+
+def test_dynamic_weighted_majority():
+    stream = SEAGenerator(1, noise_percentage=0.067, random_state=112)
+    stream.prepare_for_use()
+
+    learner = DynamicWeightedMajority(3, NaiveBayes(), beta=0.5, theta=0.01)
+
+    cnt = 0
+    max_samples = 5000
+    predictions = []
+    wait_samples = 100
+    correct_predictions = 0
+    first = True
+
+    while cnt < max_samples:
+        X, y = stream.next_sample()
+        # Test every n samples
+        if (cnt % wait_samples == 0) and (cnt != 0):
+            predictions.append(learner.predict(X)[0])
+            if y[0] == predictions[-1]:
+                correct_predictions += 1
+        if first:
+            learner.partial_fit(X, y, classes=stream.target_values)
+            first = False
+        else:
+            learner.partial_fit(X, y)
+        cnt += 1
+    performance = correct_predictions / len(predictions)
+    expected_predictions = [1, 0, 1, 1, 0, 1, 0, 1, 0, 1, 0, 0, 1, 0, 1, 1, 1,
+                            0, 1, 0, 0, 1, 0, 0, 1, 1, 1, 1, 0, 1, 0, 1, 1, 1,
+                            0, 0, 1, 1, 1, 0, 0, 1, 1, 0, 1, 0, 0, 1, 1]
+    expected_correct_predictions = 44
+    expected_performance = 0.8979591836734694
+
```

Comment:
Given that the method implements `predict_proba` please consider adding an explicit assertion for expected predicted probabilities.

Although `predict_proba` is called by `predict`, there could be cases where changes in `predict_proba` could be missed without an explicit test.

See for reference: https://github.com/scikit-multiflow/scikit-multiflow/blob/c4dbbb70d4ed839d95a18ca799f073ac9ff9ba49/tests/bayes/test_naive_bayes.py#L44-L46

Reply:
The ```predict_proba``` was a mistake, it was just to be used internally since the paper doesn't actually define what the class probability distributions would be.

I've addressed this in the most recent commit.